.github/workflows/integrationTest.yml

```diff
@@ -971,4 +982,80 @@ jobs:
 #          max_attempts: 3
 #          timeout_minutes: 8
 #          retry_wait_seconds: 5
-#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
\ No newline at end of file
+#          command: cd terraform/ec2/linux && terraform destroy --auto-approve
+
+  EC2AcceptanceIntegrationTest:
+    needs: [ECSFargateIntegrationTest, ECSEC2IntegrationTest, EC2LinuxIntegrationTest, GenerateTestMatrix]
```

Comment:
I did it initially because when devs do testing, it's most likely focused on feature testing. Acceptance tests are a lot less likely to fail due to most of our changes, and didn't wanna fit too many stuff in the first step and potentially make feature tests wait for acceptance to finish